### PR TITLE
Update Gang Up flanking rules for Remaster

### PR DESCRIPTION
**Pull request by Drental**
did my best in reading everything and handling it appropriately.

### CODE REVIEW

File: packs/feats/gang-up.json [hunk 1]
```diff
@@ -27,9 +27,9 @@
         "rules": [
             {
                 "key": "ActiveEffectLike",
-                "mode": "add",
-                "path": "system.attributes.flanking.canGangUp",
-                "value": 1
+                "mode": "override",
+                "path": "system.attributes.flanking.trueGangUp",
+                "value": true
```
[the remark this answers is not in the document]
Comment (Drental) [reply packs/feats/gang-up.json]: looked deeper into ae-like and now I understand why I got a validation error there and how not to get the validation error!
The system is amazing 😁

New version up and tested again.

File: src/module/actor/data/base.ts [hunk 2]
```diff
@@ -133,7 +133,9 @@ type GangUpCircumstance =
     /** Requires at least `number` allies within melee reach of the target */
     | number
     /** Requires the actor's animal companion to be adjacent to the target */
-    | "animal-companion";
+    | "animal-companion"
+    /** The Gang Up rogue feat allows allies to flank with the gang-upper */
+    | boolean;
```
Comment (stwlam) [inline src/module/actor/data/base.ts:138]:
```suggestion
    | true;
```

File: src/module/canvas/token/object.ts [hunk 3]
```diff
@@ -165,7 +165,15 @@ class TokenPF2e<TDocument extends TokenDocumentPF2e = TokenDocumentPF2e> extends
...
-        const gangingUp = flanking.canGangUp.some((g) => typeof g === "number" && g <= flankingBuddies.length);
+        const gangingUp =
+            flanking.canGangUp.some(
+                (g) =>
+                    (typeof g === "number" && g <= flankingBuddies.length) ||
+                    (g === true && flankingBuddies.length >= 1),
+            ) ||
+            flankingBuddies.some((b) => {
+                return b.actor?.attributes?.flanking?.canGangUp.some((g) => g === true);
+            });
```
Comment (stwlam) [inline src/module/canvas/token/object.ts:176]:
```suggestion
        const gangingUp =
            flanking.canGangUp.some(
                (g) =>
                    (typeof g === "number" && g <= flankingBuddies.length) ||
                    (g === true && flankingBuddies.length >= 1),
            ) || flankingBuddies.some((b) => b.actor?.attributes.flanking.canGangUp.some((g) => g === true));
```

File: src/module/migration/migrations/908-true-gang-up.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import { AELikeSource } from "@module/rules/rule-element/ae-like.ts";
+
+/** Add roll options to abilities allowing one to ignore the flat-footed condition from being flanked */
+export class Migration908TrueGangUp extends MigrationBase {
+    static override version = 0.908;
+
+    override async updateItem(source: ItemSourcePF2e): Promise<void> {
+        const slug = source.system.slug ?? "";
+        if (slug === "gang-up" && source.type === "feat") {
+            source.system.rules = [this.gangUp];
```
Comment (stwlam) [inline src/module/migration/migrations/908-true-gang-up.ts:12]:
```suggestion
            source.system.rules = [this.#gangUp];
```

File: src/module/migration/migrations/908-true-gang-up.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        }
+    }
+
+    /** Instead of merely shrugging the flat--footed condition, this will suppress all benefits of flanking */
+
+    private get gangUp(): AELikeSource {
```
Comment (stwlam) [inline src/module/migration/migrations/908-true-gang-up.ts:18]:
```suggestion
    get #gangUp(): AELikeSource {
```

**Pull request merged**